pkg/ssh/manifest.json.in

```diff
@@ -0,0 +1,15 @@
+{
+    "version": "@VERSION@",
+    "requires": {
+	"cockpit": "134.x"
+    },
+    "priority": 1,
+    "bridges": [
+        {
+            "match": { "host": null },
+            "spawn": [ "@libexecdir@/cockpit-ssh" ],
+            "environ": [ "SSH_ASKPASS=@libexecdir@/cockpit-askpass" ],
```

Comment:
Ack, if we don't propagate the var then it's all good, and everything else is stuff that  we can easily change later on if necessary. Thanks!